tests/proto/expected

```diff
@@ -5,6 +5,6 @@
 -rw----     1.0 fat        0 b-  0% stor 20100101.000000 anx/proto/zero/
 -rw----     2.0 fat    15505 b-  0% stor 20100101.000002 anx/proto/zero/Zero$.class
 -rw----     2.0 fat     4453 b-  0% stor 20100101.000002 anx/proto/zero/Zero$ZeroLens.class
--rw----     2.0 fat    20711 b-  0% stor 20100101.000002 anx/proto/zero/Zero.class
```

Comment:
I think that changing these numbers is safe, as that from what I've been able to gather it's the file size.
Can someone confirm or deny this, both in regards to it being safe to change and that it's the file size.